Assets/UI/Panels/citypanel.lua

```diff
@@ -1615,37 +1795,41 @@ function Initialize()
     Events.DistrictDamageChanged  .Add( OnCityProductionChanged );
     Events.LocalPlayerTurnBegin   .Add( OnLocalPlayerTurnBegin );
     Events.ImprovementChanged     .Add( OnCityProductionChanged );
+    -- ==== CQUI CUSTOMIZATION BEGIN ====================================================================================== --
     Events.InterfaceModeChanged   .Add( CQUI_OnInterfaceModeChanged );
+    -- ==== CQUI CUSTOMIZATION END ======================================================================================== --
     Events.LocalPlayerChanged     .Add( OnLocalPlayerChanged );
     Events.UnitSelectionChanged   .Add( OnUnitSelectionChanged );
     Events.PlayerResourceChanged  .Add( OnPlayerResourceChanged );
+    -- ==== CQUI CUSTOMIZATION BEGIN ====================================================================================== --
     Events.LoadScreenClose        .Add( CQUI_OnLoadScreenClose );
     Events.PlotYieldChanged       .Add( OnTileImproved );
     Events.PlayerTurnActivated    .Add( RefreshOnTurnRoll );
+    -- ==== CQUI CUSTOMIZATION END ======================================================================================== --
 
     -- LUA Events
     LuaEvents.CityPanelOverview_CloseButton.Add( OnCloseOverviewPanel );
+    LuaEvents.CityPanel_SetOverViewState.Add(    OnCityPanelSetOverViewState );
```

Comment:
the inline functions that had previously been declared at line 1633 and 1637 in the LEFT side are now "regular" functions (see note above)